Userland/DevTools/Profiler/Profile.cpp

```diff
@@ -211,6 +211,8 @@ void Profile::rebuild_tree()
     m_model->invalidate();
 }
 
+Optional<MappedObject> g_kernel_debuginfo_object;
```

Comment:
Any reason for not storing this in Profile? Profile is the one initialising it and DisassemblyModel can access it via `m_profile`

Reply:
I tried to follow how we already deal with libraries, as they are stored in a global variable (`g_mapped_object_cache`).

Theoretically, if we were to implement support for viewing multiple profile files side-by-side, storing a separate instance of the kernel's `MappedObject` in each one wouldn't make sense.